lib/Rx/Observable.php

```diff
@@ -616,8 +831,13 @@ public function scan(callable $accumulator, $seed = null)
 
     /**
      * Creates an array from an observable sequence.
```

Comment:
"Creates an observable sequence containing a single element which is an array containing all the elements of the source sequence."